package.json

```diff
@@ -14,21 +14,21 @@
     "@types/lodash": "^4.14.158",
     "@types/node": "^12.12.53",
     "@types/react-dom": "^16.9.0",
-    "axios": "^0.19.2",
+    "axios": "^0.21.1",
     "date-fns": "^2.15.0",
     "formik": "^2.1.5",
     "jwt-decode": "^2.2.0",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "^3.4.3",
+    "react-scripts": "^4.0.3",
     "typescript": "~3.7.2",
     "yup": "^0.29.3"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test --env=jest-environment-jsdom-sixteen",
+    "test": "react-scripts test --env=jest-environment-jsdom-sixteen --resetMocks=false",
```

Comment:
The React Script upgrade had a breaking change that changes the resetMocks default from false to true.  This breaks a bunch of the tests so it needs to be explicitly set back to false.